client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -20,7 +20,18 @@ const Dashboard = (): JSX.Element => {
   if (!loggedInUser) {
     history.push('/login');
 
-    return <CircularProgress />;
+    return (
+      <Grid spacing={4} container justify="center" className={classes.loadingScreen}>
+        <Grid item>
+          <CircularProgress size={150} />
+        </Grid>
+        <Grid item>
+          <Typography variant="h4" className={classes.loadingScreenText}>
+            Kanban
+          </Typography>
+        </Grid>
+      </Grid>
```

Comment:
0Instead of copying these styles & JSX, I suggest moving this to it's own component and just render that component instead of having duplicated code when we need to show a loading state.

Reply:
Sure, that's nice.